/**:
  ros__parameters:
    mass: 3.3981
    max_thrust: 230.0 #195.198
    pid_trans_p: "15.0, 15.0, 10.0"
    pid_trans_i: "2.0, 2.0, 2.0"
    pid_trans_d: "15.0, 15.0, 15.0"
    pid_rot_p: "10.0, 10.0, 10.0"
    pid_rot_i: "0.5, 0.5, 0.5"
    pid_rot_d: "4.0, 4.0, 4.0"
    select_controller: 2
    d_position: "0.0, 0.0, 1.0"
    d_rot_euler: "0.0, 0.0, 0.0"
    # anti_windup_trans: "0.0, 0.0, 0.0"
    # anti_windup_rot: "0.0, 0.0, 0.0"
    bat_discharge_a: 0.0
    bat_discharge_b: 0.0